### ISSUE TITLE
extension never fully initializes after SqlToolsService install

### ISSUE THREAD

**Issue by aligature** [microsoft/vscode-mssql#17127]
- MSSQL Extension Version: 1.11.1
- VSCode Version: 1.62.3 (user setup)
- OS Version: 10.0.18363

Steps to Reproduce:

1. Install the extension

When I installed the extension I saw logging indicating that the sqltoolsservice was being downloaded and installed. I saw no error code as a result. The lack of error code may make this a different issue than #16989.  However, every time I launch VSCode and run a query I get the error "command 'mssql.runQuery' not found".

This log file contains this message that I also see in the GUI.
`C:\Users\<my user>\AppData\Roaming\Code\logs\20211122T092529\exthost1\output_logging_20211122T092535\4-SqlToolsService Initialization.log`

```
Initializing SQL tools service for the mssql extension.
Note: mssql commands will be available after installing the service.

Platform: win32, x86_64 (Windows)
```

**Comment by cssuh** [comment]
Hi @aligature, how long after opening the extension do you try running a query?

**Comment by cssuh** [comment]
To get you unblocked, you can try:
1. Download sqltoolsservice from here:
https://azuredatastudiobuilds.blob.core.windows.net/sqltoolsservice/v3.0.0-release.139/Microsoft.SqlTools.ServiceLayer-win-x64-net5.0.zip
2. Unzip the files to: 
%USERPROFILE%\.vscode\extensions\ms-mssql.mssql-1.11.1\sqltoolsservice\3.0.0-release.139\Windows

**Comment by aligature** [comment]
@cssuh thanks for the help.

I don't think it's a proxy issue since the download appears to succeed and the unzip happens. I actually already had the files at the `%USERPROFILE%\.vscode\extensions\ms-mssql.mssql-1.11.1\sqltoolsservice\3.0.0-release.139\Windows` location dated yesterday. However I still get the "command 'mssql.runQuery' not found" error even after waiting many minutes.

I checked for open handles in that extension path and I see that VSCode is running two subprocesses from there "MicrosoftSqlToolsServiceLayer.exe" and "SqlToolsResourceProviderService.exe".

**Comment by kburtram** [comment]
@aligature from the description of the issue I'm not sure if this is related to SQL Tools Service (as you're not seeing an error message from that component).  Do you see any errors from the extension in the Hel->Developer Tools Console?  Also, sometimes if there is invalid JSON in the user settings file this cause the extension to fail to load.  Could you please use "Open Settings (JSON)" command and verify there aren't any errors in the JSON content there?  Thanks!

**Comment by aligature** [comment]
This problem has resolved itself after the recent install of v1.12.0. Unfortunately I didn't get a chance to look at the developer console log in the failing state. The issue must not have been invalid json since the new version works and my settings are unchanged.

**Comment by kburtram** [comment]
@aligature thanks for following-up.  Sorry for the inconvenience of encountering this bug.
